airgun/widgets.py

```diff
@@ -490,6 +491,54 @@ def fill(self, value):
         return self.select(checkbox_locator, checkbox_value)
 
 
+class LimitInput(Widget):
+    """Input for managing limits (e.g. Hosts limit). Consists of 'Unlimited'
+    checkbox and text input for specifying the limit, which is only visible if
+    checkbox is unchecked.
+
+    Example html representation::
+
+        <input type="checkbox" name="limit"
+         ng-model="activationKey.unlimited_hosts"...>
+
+        <input id="max_hosts" name="max_hosts"
+         ng-model="activationKey.max_hosts"
+         ng-required="!activationKey.unlimited_hosts" type="number" min="1"
+         max="2147483648"...>
+
```

Comment:
missed mandatory section
```
     Locator example::

```

Reply:
As there's no locator. And can't be any - 2 actual widgets inside are located in different parts of DOM, so if we were accepting locators, we'd probably had to go with smth like `LimitInput(locator1='foo', locator2='bar')`. That was my initial concern i've told you, that's why i'm not really happy about such approach.

Reply:
cool, add that information to that section

Reply:
done